docs/tutorials/MenuGuide.md

```diff
@@ -0,0 +1,237 @@
+Creating a simple Menu
+======================
+
+### From open.mp Wiki
+
+
+
+Contents
+--------
+
+*   1 Menus in Pawn
+*   2 First menu steps
+*   3 Add some menu items
+*   4 Creating the effects for the items
+*   5 Last steps
+*   6 Last words
+*   7 A full test script for menus
+
+
+### 1 Menus in Pawn
+-------------
+
+Menus look very complicated and difficult to script for the most players, although it isn't. Here I will show you how to create a simple menu. At the end we will have created a teleport menu.
+
+
+### 2 First menu steps
+----------------
+
+First we have to create a menu. The prefix before is Menu: this makes the variable the correct type \[there are different types for different uses such as Float: bool: Text3D: etc\] Insert this line after the #includes, #defines, forwards etc:
```

Comment:
https://www.open.mp/docs/meta/Contributing#use-code-snippets-for-technical-references